frontend/src/app/components/containers/complaints/complaint-filter-bar.tsx

```diff
@@ -5,13 +5,16 @@ import { clearFilter } from "../../../store/reducers/complaint-filters";
 import { ComplaintFilters } from "../../../types/complaints/complaint-filters/complaint-filters";
 import MapListToggle from "../../common/map-list-toggle";
 import SearchInput from "../../common/search-input";
+import { Button } from "react-bootstrap";
 
 type Props = {
   toggleViewType: (view: "map" | "list") => void;
   viewType: "map" | "list";
   complaintType: string;
   searchQuery: string | undefined;
   applySearchQuery: Function;
+  toggleShowMobileFilters: any;
```

Comment:
We want to try and avoid the use of the `any` type if we can here and instead use an explicit type

Reply:
Updated as requested.